src/App.jsx

```diff
@@ -1,98 +1,11 @@
-/* eslint-disable max-len */
-
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import movies from './api/movies.json';
+import { MovieList } from './components/MovieList';
```

Comment:
The imported component 'MovieList' is spelled incorrectly. It should be 'MoviesList' as per the task description.